cmd/root.go

```diff
@@ -121,67 +124,67 @@ func checkIfValidAnalyzer(_ []string) error {
 	return nil
 }
 
-func getPrepperForImage(image string) (pkgutil.Prepper, error) {
-	cli, err := client.NewEnvClient()
-	if err != nil {
-		return nil, err
-	}
-
-	if pkgutil.IsTar(image) {
-		return &pkgutil.TarPrepper{
-			Source: filepath.Clean(image),
-			Client: cli,
-		}, nil
+func getImageForName(imageName string) (pkgutil.Image, error) {
+	if !pkgutil.IsTar(imageName) && !pkgutil.HasTag(imageName) {
+		imageName = imageName + pkgutil.LatestTag
```

Comment:
Why do you have to add latest? WeakValidation should add that below.

Reply:
ah you're right, removed